app/models/printer.rb

```diff
@@ -1,7 +1,7 @@
 class Printer < ApplicationRecord
   CLASS_TYPE_PRINTER_ALIASES = {
     'TubeRack' => 'Plate',
-    'Tube' => 'SampleTube'
+    'SampleTube' => 'Tube'
```

Comment:
Whoops. Yep, got that one the wrong way round. Weird think is, I thought it seemed an odd way round when I did it.